docs/make.jl

```diff
@@ -22,6 +22,9 @@ makedocs(
         "Introduction" => "index.md",
         "User Guide" => Any[
             "Getting Started" => "man/getting_started.md",
+            "Working with DataFrames" => "man/Working_with_DataFrames.md",
+            "Replacing Data" => "man/replacing_data.md",
```

Comment:
do you feel (as a user) it is worth to promote "Replacing Data" to top level here? Or it is better to have it as a section in "Working with DataFrames"?

Reply:
I don't think so because when a user will review 'Working with DataFrames' then after completing that section he can directly jump to the 'Replacing Data' section in the series. But both things will be fine(show it as a section in 'working with DataFrames' or promote it to the top level).

Reply:
I would then include "Replacing Data" in "Working with DataFrames"